### PR TITLE
Tcf day of

### PR DESCRIPTION
**Pull request by wgervasio**
add /tcf/day-of
modify a few typos in aristanetoworks.md, safesoftware.md and activisiondemonware.md
add several diagrams and pdfs to /files/tcf2024/

### CODE REVIEW

**Comment by kewbish** [comment]
Is this ready to be merged?

**Comment by wgervasio** [comment]
nope, will ping when it's good!

**Comment by wgervasio** [comment]
Ready! @kewbish

**Comment by wgervasio** [comment]
most likely broke several style rules in that file

File: themes/hugo-bootstrap-5/layouts/tcf/tcf-day-of.html [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+        <h2 id="booth-assignments" style="text-align: center;">
+          Booth Assignments
+        </h2>
+        <p style="text-align: center;">
+          The following booth assignments are subject to change. Check back on
+          the day for the final assignments
+        </p>
+        <div class="grid-container">
+          <div class="row pt-4">
+            {{ $.Scratch.Set "counter" 0 }}
```
Comment (wgervasio) [inline themes/hugo-bootstrap-5/layouts/tcf/tcf-day-of.html:284]: this is a temporary fix. I will add to the company markdowns the true venue numbers next week. Rn is not the time

File: content/tcf/activisiondemonware.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: tcf-company-2024
 title: Activision/Demonware
 link: https://www.activision.com/
 short_bio: >
-  Headquartered in Santa Monica, California, Activision Blizzard (NASDAQ: ATVI) is one the world's largest and most successful interactive entertainment companies and sits at the intersection of media, technology, and entertainment. Activision Blizzard is a member of the S&P 500, and has operations throughout the world. The company has over 6,000 employees, and its games are played in 196 countries.At the core of Arista's platform is the Extensible Operating System (EOS™), a groundbreaking network operating system with single-image consistency across hardware platforms, and modern core architecture enabling in-service upgrades and application extensibility.
```
Comment (kewbish) [inline content/tcf/activisiondemonware.md:6]: Good catch :skull:

**Review by kewbish — APPROVED** [review]
Will approve to unblock, I'll let you merge if you want to make changes re. Youssef's edits!

**Comment by wgervasio** [comment]
merci!

**Pull request merged**